pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -32,28 +32,31 @@
           </p>
           <% } %>
 
-          <% if (locals.assessment.require_honor_code) { %>
-          <div class="card card-secondary mb-4 test-class-honor-code">
-            <ul class="list-group list-group-flush">
-              <li class="list-group-item py-2">I certify that I am <%= user.name %> and I am allowed to take this assessment.</li>
-              <li class="list-group-item py-2">I pledge on my honor that I will not give or receive any unauthorized assistance on this assessment and that all work will be my own.</li>
-            </ul>
+          <% if (locals.assessment.group_work) { %>
+            <%- include('../partials/studentGroupControls.ejs', {groupsize: groupsize}); %>
+          <% } else { %>
+            <% if (locals.assessment.require_honor_code) { %>
```

Comment:
It looks like this assumes that the honor code and group work are mutually exclusive. I'm not 100% sure if that's desirable or not, but I want to make sure we're thinking through that deliberately.

Reply:
This condition is intentional because in` partials/studentGroupControls.ejs` I have a check for student honor code, but I change the language to be plural ("We pledge on our honor...") for groups. If that needs to be reworked for better readability or otherwise, happy to do so!

Reply:
Ah! I didn't realize that `studentGroupControls` included the prompt too. I think this is ok.